abnf/odata-aggregation-testcases.yaml

```diff
@@ -266,7 +275,7 @@ TestCases:
 
   - Name: aggregate - countdistinct
     Rule: queryOptions
-    Input: $apply=aggregate(Product with countdistinct as DistinctProducts)
+    Input: $apply=aggregate(Product/Name with countdistinct as DistinctProducts)
```

Comment:
I assume this was by mistake: `Product` is a navigation property, not a primitive property.